add a note on IdP Discovery feature limitation (#2064)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md

The Okta Sign-In Widget (version 2.5.0 or above) interacts with the IdP Discovery policy to redirect end users as needed to the IdP determined by the policy.

> > **Note:** v1 API supports IdP Discovery in Okta-hosted Widget setup only.

Reviewer: ```suggestion
> **Note:** v1 of the Okta API supports IdP Discovery with the Okta-hosted Widget only.
```
